app/src/main/java/pl/agh/eye/CameraActivity.kt

```diff
@@ -55,13 +80,48 @@ class CameraActivity : AppCompatActivity(), CvCameraViewListener2 {
         }
     }
 
+    private fun getClassifier(name: String, cascadeResource: Int): CascadeClassifier {
+        val inputStream: InputStream = resources.openRawResource(
+            cascadeResource
+        )
+        val cascadeDir: File = getDir("cascade", Context.MODE_PRIVATE)
+        val mCascadeFile = File(
+            cascadeDir,
+            name
+        )
+        val outputStream = FileOutputStream(mCascadeFile)
+
+        val buffer = ByteArray(4096)
+        var bytesRead: Int
+        while (inputStream.read(buffer).also { bytesRead = it } != -1) {
```

Comment:
Dałoby radę możę metodę `File.copyTo` użyć? Tam też można wyspefycikować rozmiar bufora: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/java.io.-file/copy-to.html